RestaurantApp/CustomControlls/ButtonFrame.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace RestaurantApp.CustomControlls
+{
+    public class ButtonFrame : Frame
```

Comment:
you called it 'buttonframe' but actually you can use it not only for buttons, so this name doesnt explain usage of the control